### PR TITLE
Remove FAQ section on github branch :)

### PR DESCRIPTION
**Pull request by Mariatta**
Move the following items from FAQ:
- disagreement with issue resolution -> tracker.rst
- regenerating configure -> setup.rst
- porting python to a new platform -> porting.rst (new file)

In index.rst:
- change any reference to FAQ into Help.
- remove faq from TOC
- add porting to TOC

Closes https://github.com/python/devguide/issues/27

### CODE REVIEW

File: tracker.rst [hunk 1]
```diff
@@ -136,6 +136,23 @@ In the sidebar you can also find links to summaries for easy issues and
...
+Disagreement With a Resolution on the Issue Tracker
+===================================================
+
+First, take some time to consider any comments made in association with the
+resolution of the tracker issue. On reflection, they may seem more reasonable
+than they first appeared.
+
+If you still feel the resolution is incorrect, then raise the question on
+`python-dev`_. If the consensus there supports the disputed resolution, please
+take any further objections to `python-ideas`_ (or some other forum). Further
```
Comment (brettcannon) [inline tracker.rst]: If python-dev has already said that the resolution was right, then sending them to python-ideas to continue to vent isn't productive. Also mention that people should not re-open an issue once a core developer has closed it.

Comment (Mariatta) [inline tracker.rst]: Thanks @brettcannon :) I updated this section.

File: tracker.rst [hunk 2]
```diff
@@ -136,6 +136,22 @@ In the sidebar you can also find links to summaries for easy issues and
...
+
+First, take some time to consider any comments made in association with the
+resolution of the tracker issue. On reflection, they may seem more reasonable
+than they first appeared.
+
+If you still feel the resolution is incorrect, then raise the question on
+`python-dev`_. Further argument on `python-dev`_ after a consensus has been
+reached amongst the core developers is unlikely to win any converts.
+
+You should not reopen issues that have been closed by a core developer.
```
Comment (willingc) [inline tracker.rst]: @mariatta This is fine as is. A slight refinement might be more direct and welcoming: `Issues closed by a core developer have already been carefully considered. Please do not reopen a closed issue.`

**Comment by Mariatta** [comment]
Thanks @willingc I updated it based on your suggestion :)

**Comment by brettcannon** [comment]
thanks!

**Pull request merged**